lisa/transformers/kernel_installer.py

```diff
@@ -140,37 +142,14 @@ def _internal_run(self) -> Dict[str, Any]:
             if installer.runbook.source != "linux-image-azure-fde":
```

Comment:
we should remove if

Reply:
That step is not needed for cvm. So lets keep it.